data/news/san_mateo.json

```diff
@@ -3,6 +3,13 @@
   "title": "San Mateo County COVID-19 News",
   "home_page_url": "https://cmo.smcgov.org/press-releases",
   "items": [
+    {
+      "id": "12211 at https://cmo.smcgov.org",
+      "url": "https://cmo.smcgov.org/press-release/aug-1-2020-update-san-mateo-countys-covid-19-monitoring-list-status",
+      "title": "Aug. 1, 2020-Update on San Mateo County's COVID-19 Monitoring List Status",
+      "date_published": "2020-08-01T20:30:27Z",
+      "summary": "&amp;nbsp;The County of San Mateo remains in discussion with the state and as of Aug. 1 our status on the COVID-19 Monitoring List remains the same,&amp;nbsp;without&amp;nbsp;the additional business closures being required. We will inform businesses and public when we have confirmation of any status change.\n\nInformation about the County of San Mateo\u2019s response to COVID-19 is at&amp;nbsp;www.smcgov.org.\n\n&amp;nbsp;\n\n###"
```

Comment:
This double-encoded HTML entities are actually in the source data, so we’re not messing up here. I’m going to see if I can do a little in the scraper code to clean them up, though.